input/postgres/statements.go

```diff
@@ -117,10 +101,59 @@ func GetStatements(server *state.Server, logger *util.Logger, db *sql.DB, global
 				" the monitoring helper functions (https://github.com/pganalyze/collector#setting-up-a-restricted-monitoring-user)" +
 				" or connect as superuser, to get query statistics for all roles.")
 		}
+
+		var extSchema string
+		var foundExtVersion float32
+		err = db.QueryRow(QueryMarkerSQL+"SELECT nspname, extversion FROM pg_extension pge INNER JOIN pg_namespace pgn ON pge.extnamespace = pgn.oid WHERE pge.extname = 'pg_stat_statements'").Scan(&extSchema, &foundExtVersion)
```

Comment:
IIRC, this will fail if more than one row is returned (that is: the extension is installed more than once). I'm not sure if that's possible; just trying to think through edge cases.